tests/WebServer.php

```diff
@@ -70,6 +71,13 @@ final class WebServer
         'datadog.trace.client_ip_header_disabled' => 'true',
     ];
 
+    /**
+     * Persisted apache instance for the life time of the testsuite - we use reload instead of restart to apply changes
```

Comment:
Maybe leave a specific note about the lifecycle bugs to answer "why use reload instead of restart?"